dfget/core/api/supernode_api.go

```diff
@@ -60,16 +62,36 @@ type supernodeAPI struct {
 
 var _ SupernodeAPI = &supernodeAPI{}
 
+//dfget --node ends with ":port" or not
+func (api *supernodeAPI) nodeToIPAndPort( node string) (ip string, port int ){
```

Comment:
Could you please write a unit test for this function?

Reply:
OK. I have done it.